src/Raven.Client/Server/Operations/ConfigureExpirationBundleOperation.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Session;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Json.Converters;
+using Raven.Client.Server.expiration;
+using Sparrow.Json;
+
+namespace Raven.Client.Server.Operations
+{
+    public class ConfigureExpirationBundleOperation : IServerOperation<ConfigureExpirationBundleOperationResult>
```

Comment:
@talweiss1982 you can move this operation to tests if you need it, I have couple of more 'internal' operations/commands in tests e.g. https://github.com/ravendb/ravendb/blob/v4.0/test/Tests.Infrastructure/CreateSampleDataOperation.cs